secrets/secrets.go

```diff
@@ -0,0 +1,69 @@
+package secrets
+
+import (
+	"errors"
+	"sync"
+)
+
+var (
+	// ErrNotImplemented default secrets in OSD
+	ErrNotImplemented = errors.New("Not Implemented")
```

Comment:
Add something like `ErrAccessDenied` returned by `Login()` when the credentials are not accepted. Then this valued can be checked for in the login function in the REST server to see it `Login()` returned an error because access was denied or some other reason. If any other reason other than ErrAccessDenied, then the code should return http.statusinteralerror.

Also, please coordinate this error with those you implemented the Login() function in Px. it may be they already have an error value we could use here.

Reply:
I think ErrUnauthorized/ErrNotAuthenticated is more precise than AccessDenied in case of a login. AccessDenied is more like a 403 (Forbidden) which should be thrown on accessing a certain resource.